dev/io.openliberty.restfulWS.3.0_fat/bnd.bnd

```diff
@@ -19,8 +19,10 @@ fat.project: true
 
 
 -buildpath: \
-    io.openliberty.jakarta.annotation.2.0;version=latest,\
-    io.openliberty.jakarta.jaxb.3.0;version=latest,\
-    io.openliberty.jakarta.restfulWS.3.0;version=latest,\
+	io.openliberty.jakarta.annotation.2.0;version=latest,\
+	io.openliberty.jakarta.jaxb.3.0;version=latest,\
+	io.openliberty.jakarta.validation.3.0;version=latest,\
+	io.openliberty.jakarta.restfulWS.3.0;version=latest,\
 	io.openliberty.jakarta.servlet.5.0;version=latest,\
-	com.ibm.ws.componenttest.2.0
+	com.ibm.ws.componenttest.2.0,\
+	jakarta.validation.jakarta.validation-api
```

Comment:
You shouldn't need this line since you have `io.openliberty.jakarta.validation.3.0;version=latest,\` above.  IIUC, there is a build time cost for depending on artifacts in Artifactory/Maven Central/etc. vs depending on artifacts built during the build.  The latter is preferred.